jobs/sync-upstream/sync.py

```diff
@@ -156,22 +172,23 @@ def _rename_branch(
                     continue
 
             if not params.get("supports_rename", True):
-                log.info(f"Skipping {layer_name} :: does not support branch renaming")
+                log.info(
+                    f"Skipping  :: {layer_name:^40} :: does not support branch renaming"
+                )
                 continue
 
             repo = Repository.with_session(*downstream.split("/"), read_only=dry_run)
-            log.info(
-                f"Renaming Branch:: {layer_name:^35} :: from: {from_name} to:{to_name}"
-            )
 
             if from_name not in repo.branches:
-                log.info(f"Skipping  :: {layer_name:^35} :: {from_name} doesn't exist")
+                log.info(f"Skipping  :: {layer_name:^40} :: {from_name} doesn't exist")
                 continue
 
             if to_name in repo.branches:
-                log.info(f"Skipping  :: {layer_name:^35} :: {to_name} already exists")
+                log.info(f"Skipping  :: {layer_name:^40} :: {to_name} already exists")
                 continue
 
+            log.info(f"Renaming :: {layer_name:^40} :: from: {from_name} to:{to_name}")
```

Comment:
```suggestion
            log.info(f"Renaming  :: {layer_name:^40} :: from: {from_name} to:{to_name}")
```
Small formatting fix to get all the :: lined up

Reply:
you have some sort of eagle eyes 🦅